src/components/Owner/BarCustomerList.jsx

```diff
@@ -1,32 +1,18 @@
 import React, { useState, useEffect } from 'react';
-import { Grid, Typography } from '@material-ui/core';
+import { Grid, Typography, Divider, Paper } from '@material-ui/core';
 import axios from 'axios';
-import CustomerEntry from './CustomerEntry.jsx'
-
-const BarList = ({ barId, setCount }) => {
-    const [customerList, setCustomerList] = useState([]);
-
-    //TODO: Check out setInterval to grabbing the list of customers
-    const fetchCustomers = () => {
-        axios.get(`/db/cb/list?barId=${barId}`)
-            .then(({ data }) => {
-                setCustomerList(data);
-                setCount(data.length);
-            })
-    }
-
-    useEffect(() => {
-        fetchCustomers();
-        setTimeout(() => {
-            fetchCustomers();
-        }, 2000)
-    }, [])
+import CustomerEntry from './CustomerEntry.jsx';
+import PhoneIcon from '@material-ui/icons/Phone';
+import PlusOneIcon from '@material-ui/icons/PlusOne';
 
+const BarList = ({ customerList, barId }) => {
     if (customerList.length === 0) {
         return (
             <Grid container direction="column" justify="center" alignItems="center">
```

Comment:
move all of the rendered logic to a separate component and then pass down props where needed